forloop_modules/function_handlers/database_handlers.py

```diff
@@ -1066,17 +1073,13 @@ def direct_execute(self, db_name_origin, db_name_destination, migration_name="mi
         db_connection_destination.test_database_connection()
 
         if db_connection_origin is None or db_connection_destination is None:
-            #glc.show_warning_popup_message("Db connection not found")
```

Comment:
Do not remove this - please put it back everywhere, it still is a splace which should trigger a warning popup we just should do it differently than with GLC later